addons/dialogue_manager/l10n/zh-cn.po

```diff
@@ -0,0 +1,403 @@
+msgstr "如果你使用静态键，这将会帮助你寻找未添加至翻译文件的键。"
+
+msgid "settings.wrap_long_lines"
+msgstr "自动折行"
+
+msgid "settings.include_failed_responses"
+msgstr "在判断条件失败时仍显示选项"
+
+msgid "settings.ignore_missing_state_values"
+msgstr "忽略全局变量缺失错误（不建议）"
+
+msgid "settings.custom_test_scene"
+msgstr "自定义测试场景（必须继承自BaseDialogueTestScene）"
+
+msgid "settings.states_shortcuts"
+msgstr "全局变量映射"
+
+msgid "settings.states_message"
+msgstr "当一个 Autoload 在这里被勾选，他的所有成员会被映射为全局变量。"
+
+msgid "settings.states_hint"
+msgstr "比如，当你开启对于“Foo”的映射时，你可以将“Foo.bar”简写成“bar”。"
+
+msgid "n_of_n"
+msgstr "第{index}个，共{total}个"
+
+msgid "search.previous"
+msgstr "查找上一个"
+
+msgid "search.next"
+msgstr "查找下一个"
+
+msgid "search.match_case"
+msgstr "大小写敏感"
+
+msgid "search.toggle_replace"
+msgstr "替换"
+
+msgid "search.replace_with"
+msgstr "替换为"
+
+msgid "search.replace"
+msgstr "替换"
+
+msgid "search.replace_all"
+msgstr "全部替换"
+
+msgid "files_list.filter"
+msgstr "查找文件"
+
+msgid "titles_list.filter"
+msgstr "查找标题"
+
+msgid "errors.key_not_found"
+msgstr "键“{key}”未找到"
+
+msgid "errors.line_and_message"
+msgstr "第{line}行第{colume}列发生错误：{message}"
+
+msgid "errors_in_script"
+msgstr "你的脚本中存在错误。请修复错误，然后重试。"
+
+msgid "errors_with_build"
+msgstr "请先解决 Dialogue 中的错误。"
+
+msgid "errors.import_errors"
+msgstr "被引入的文件存在问题。"
+
+msgid "errors.already_imported"
+msgstr "文件已被引入。"
+
+msgid "errors.duplicate_import"
+msgstr "引入名不能重复。"
+
+msgid "errors.empty_title"
+msgstr "标题名不能为空。"
+
+msgid "errors.duplicate_title"
+msgstr "标题名不能重复。"
+
+msgid "errors.nested_title"
+msgstr "标题不能嵌套。"
+
+msgid "errors.invalid_title_string"
+msgstr "标题名非法。"
+
+msgid "errors.invalid_title_number"
+msgstr "标题不能以数字开始。"
+
+msgid "errors.unknown_title"
+msgstr "标题未定义。"
+
+msgid "errors.jump_to_invalid_title"
+msgstr "标题名非法。"
+
+msgid "errors.title_has_no_content"
+msgstr "目标标题为空。请替换为“=> END”。"
+
+msgid "errors.invalid_expression"
+msgstr "表达式非法。"
+
+msgid "errors.unexpected_condition"
+msgstr "未知条件。"
+
+msgid "errors.duplicate_id"
+msgstr "ID 重复。"
+
+msgid "errors.missing_id"
+msgstr "ID 不存在。"
+
+msgid "errors.invalid_indentation"
+msgstr "缩进非法。"
+
+msgid "errors.condition_has_no_content"
+msgstr "条件下方不能为空。"
+
+msgid "errors.incomplete_expression"
+msgstr ""
+
+msgid "errors.invalid_expression_for_value"
+msgstr ""
+
+msgid "errors.file_not_found"
+msgstr "文件不存在。"
+
+msgid "errors.unexpected_end_of_expression"
+msgstr ""
+
+msgid "errors.unexpected_function"
+msgstr "函数不应存在。"
+
+msgid "errors.unexpected_bracket"
+msgstr "方括号不应存在。"
+
+msgid "errors.unexpected_closing_bracket"
+msgstr "方括号不应存在。"
+
+msgid "errors.missing_closing_bracket"
+msgstr "闭方括号不存在。"
+
+msgid "errors.unexpected_operator"
+msgstr "操作符不应存在。"
+
+msgid "errors.unexpected_comma"
+msgstr "逗号不应存在。"
+
+msgid "errors.unexpected_colon"
+msgstr "冒号不应存在。"
+
+msgid "errors.unexpected_dot"
+msgstr "句号不应存在。"
+
+msgid "errors.unexpected_boolean"
+msgstr "true/false 不应存在。"
+
+msgid "errors.unexpected_string"
+msgstr ""
+
+msgid "errors.unexpected_number"
+msgstr ""
```

Comment:
This is when you do something like forget the brackets around a function arguments:

![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/78984/485ab12f-fc2e-4bf1-95d9-20a6f868de15)